packages/broker/test/integration/per-node-metrics.test.ts

```diff
@@ -81,21 +121,12 @@ describe('per-node metrics', () => {
         })
 
         storageNode = await startBroker({
-            name: 'storageNode',
+            name: 'storageNode', 
             privateKey: storageNodeAccount.privateKey,
             trackerPort,
+            wsPort,
```

Comment:
The common setup is separate the storage nodes and publishe nodes. Maybe worth considering to use the same architecture?